infra/tf/atlantis.md

```diff
@@ -0,0 +1,8 @@
+# Atlantis Truss
+
+We're beginning to use Atlantis throughout projects at Truss. You can explore the Atlantis setup in Truss' commercial accounts for the prod tiers [here](https://github.com/trussworks/legendary-waddle) and the dev tiers [here](https://github.com/trussworks/legendary-waddle-dev). There's more to come, but for now here are some helpful things to know.
+
+## Tips and Gotchas
+
+* When resetting a GitHub personal token in Parameter Store, you will have to redeploy the Atlantis instance. Otherwise, the new credential will not be updated in the instance.
```

Comment:
I suspect you actually just need to restart the atlantis task in ECS, technically.

Reply:
In fact, that is not correct! I tried that multiple times and it wasn't doing the trick--it's because of the daisy-chaining between the module-created ssm param that populates after pulling down your secret from your var and your var.